connectors/sync_job_runner.py

```diff
@@ -215,16 +216,23 @@ async def _sync_done(self, sync_status, sync_error=None):
     @with_concurrency_control()
     async def sync_starts(self):
         if not await self.reload_connector():
-            return False
+            raise SyncJobStartError(f"Couldn't reload connector {self.connector_id}")
 
         if self.connector.last_sync_status == JobStatus.IN_PROGRESS:
             logger.debug(
-                f"Connector {self.connector_id} is syncing, skip the job {self.job_id}..."
+                f"A sync job is started for connector {self.connector_id} by another connector instance, skipping..."
+            )
```

Comment:
Do you need to log like this, since the code that catches the exception probably reports error message too

Reply:
It's handled by `ConcurrentTasks`, and I don't think it properly logs anything if an exception is raised:

https://github.com/elastic/connectors-python/blob/da8a867218d90e3523624dcc6a3c278bdb767dc9/connectors/utils.py#L292

Reply:
Ha, I think it should! I'll add an issue about it, let's keep it as is for now :)